pkg/cloud/gcp/actuators/machine/actuator_test.go

```diff
@@ -1,30 +1,253 @@
 package machine
 
 import (
+	"context"
+	"fmt"
+	"path/filepath"
 	"testing"
+	"time"
 
+	. "github.com/onsi/gomega"
+	"github.com/openshift/cluster-api-provider-gcp/pkg/apis/gcpprovider/v1beta1"
+	computeservice "github.com/openshift/cluster-api-provider-gcp/pkg/cloud/gcp/actuators/services/compute"
 	machinev1 "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
+
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/kubernetes/scheme"
-	"k8s.io/client-go/tools/record"
-	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
 func init() {
 	// Add types to scheme
 	machinev1.AddToScheme(scheme.Scheme)
 }
 
-func TestActuatorCreate(t *testing.T) {
-	eventsChannel := make(chan string, 1)
-	recorder := &record.FakeRecorder{
-		Events: eventsChannel,
+func TestMachineEvents(t *testing.T) {
```

Comment:
I think this method should be called `TestActuatorScopeEvents` or just `TestActuatorEvents`

Reply:
👍